pkgs/libresoc/openpower-isa.nix

```diff
@@ -63,4 +66,30 @@ with python39Packages;
         # linker script vendored in test library
         cp ./src/openpower/simulator/memmap $out/${python39.sitePackages}/openpower/simulator/
       ''
+      + (
+        # this project's build steps do not map cleanly onto Nix's build stages.
+        # setuptools builds the the `entry_points` into binaries that are used to codegen
+        # directly into the source directory, which is then bundled into a wheel and installed.
+        #
+        # for Nix, there's a chicken-and-egg problem with this:
+        # only once the package has been built can the codegen be run to... build the package from source.
+        # as a result, the installPhase is overloaded to effectively double as the build phase,
+        # so, the majority of build steps actually occur here, at the end:
```

Comment:
maybe there's a way to model this as a bootstrapped package (e.g. pypy or chicken scheme), but this make sense here